lib/rubocop/cop/rspec/describe_method.rb

```diff
@@ -3,32 +3,123 @@
 module RuboCop
   module Cop
     module RSpec
-      # Checks that the second argument to `describe` specifies a method.
+      # Checks that arguments and sub-describes of `describe` specify a method.
       #
       # @example
       #   # bad
-      #   describe MyClass, 'do something' do
+      #   describe MyClass do
+      #     describe 'flowers' do; end
+      #   end
+      #
+      #   describe MyClass, 'flowers' do
       #   end
       #
       #   # good
+      #   describe MyClass do
+      #     describe '#my_instance_method' do; end
+      #   end
+      #
+      #   describe MyClass do
+      #     describe '.my_class_method' do; end
+      #   end
+      #
       #   describe MyClass, '#my_instance_method' do
       #   end
       #
       #   describe MyClass, '.my_class_method' do
       #   end
+      #
+      # @example configuration
+      #
+      #   # .rubocop.yml
+      #   RSpec/DescribeMethod:
+      #     IgnoredDescribes:
```

Comment:
I'm not sure how useful this configuration is. Are there instances where configuring an `IgnoreDescribes` entry would buy me more than just using `rubocop:disable` once or twice?